Doc/library/sys.rst

```diff
@@ -11,6 +11,24 @@ interpreter and to functions that interact strongly with the interpreter. It is
 always available. Unless explicitly noted otherwise, all variables are read-only.
 
 
+.. data:: abi_info
+
+   An object containing information about the ABI of the currently running
+   Python interpreter.  The following attributes are available in cpython:
+
+   *pointer_bits* is the width of pointers in bits, as an integer, equivalent
+   to ``8 * sizeof(void *)``, i.e. usually ``32`` or ``64``.
+
+   *Py_GIL_DISABLED* is a boolean indicating whether the interpreter was built
```

Comment:
I would prefer consistency between attribute names. For example, rename this one to `gil_disabled`.

I also dislike negative options. Would it make sense to rename the attribute to `gil_enabled`? If would be False in a Free Threaded build.

Reply:
> So, gil_disabled or free_threaded?

free_threaded